docs/content/routing/routers/index.md

```diff
@@ -253,6 +253,85 @@ The table below lists all the available matchers:
     For instance, `PathPrefix: /products` would match `/products` but also `/products/shoes` and `/products/shirts`.
     Since the path is forwarded as-is, your service is expected to listen on `/products`.
 
+### Priority
+
+To avoid path overlap, routes are sorted, by default, in descending order using rules length: the largest length has the highest priority.
```

Comment:
```suggestion
To avoid path overlap, routes are sorted, by default, in descending order using rules length. The priority is directly equal to the length of the rule, and so the longest length has the highest priority.
```